hostname sb039.cluster
ipconfig 1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000 link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00 inet 127.0.0.1/8 scope host lo valid_lft forever preferred_lft forever inet6 ::1/128 scope host valid_lft forever preferred_lft forever 2: enp4s0f0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc mq state UP group default qlen 1000 link/ether a0:8c:f8:74:c8:e0 brd ff:ff:ff:ff:ff:ff inet 10.3.81.119/22 brd 10.3.83.255 scope global enp4s0f0 valid_lft forever preferred_lft forever inet6 fe80::a28c:f8ff:fe74:c8e0/64 scope link valid_lft forever preferred_lft forever 3: enp4s0f1: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:74:c8:e1 brd ff:ff:ff:ff:ff:ff 4: enp4s0f2: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:74:c8:e2 brd ff:ff:ff:ff:ff:ff 5: enp4s0f3: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:74:c8:e3 brd ff:ff:ff:ff:ff:ff 6: ib0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 4092 qdisc pfifo_fast state UP group default qlen 256 link/infiniband 80:00:00:29:fe:80:00:00:00:00:00:00:38:bc:01:03:00:4c:f3:70 brd 00:ff:ff:ff:ff:12:40:1b:ff:ff:00:00:00:00:00:00:ff:ff:ff:ff inet 10.3.89.119/22 brd 10.3.91.255 scope global ib0 valid_lft forever preferred_lft forever inet 10.3.93.119/22 brd 10.3.95.255 scope global ib0:1 valid_lft forever preferred_lft forever inet6 fe80::3abc:103:4c:f370/64 scope link valid_lft forever preferred_lft forever 7: ib1: <BROADCAST,MULTICAST> mtu 4092 qdisc noop state DOWN group default qlen 256 link/infiniband 80:00:00:29:fe:80:00:00:00:00:00:00:38:bc:01:03:00:4c:f3:71 brd 00:ff:ff:ff:ff:12:40:1b:ff:ff:00:00:00:00:00:00:ff:ff:ff:ff
2024-02-01 05:54:21,578 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.3.89.119:39395'
2024-02-01 05:54:37,036 - distributed.worker - INFO -       Start worker at:    tcp://10.3.89.119:43055
2024-02-01 05:54:37,038 - distributed.worker - INFO -          Listening to:    tcp://10.3.89.119:43055
2024-02-01 05:54:37,038 - distributed.worker - INFO -           Worker name:           SLURMCluster-147
2024-02-01 05:54:37,038 - distributed.worker - INFO -          dashboard at:          10.3.89.119:41181
2024-02-01 05:54:37,038 - distributed.worker - INFO - Waiting to connect to:     tcp://10.3.88.11:37020
2024-02-01 05:54:37,038 - distributed.worker - INFO - -------------------------------------------------
2024-02-01 05:54:37,038 - distributed.worker - INFO -               Threads:                          2
2024-02-01 05:54:37,038 - distributed.worker - INFO -                Memory:                  20.49 GiB
2024-02-01 05:54:37,038 - distributed.worker - INFO -       Local Directory: /scratch/dask-worker-space/worker-9cjzqzyx
2024-02-01 05:54:37,038 - distributed.worker - INFO - -------------------------------------------------
2024-02-01 05:54:37,249 - distributed.worker - INFO -         Registered to:     tcp://10.3.88.11:37020
2024-02-01 05:54:37,249 - distributed.worker - INFO - -------------------------------------------------
2024-02-01 05:54:37,250 - distributed.core - INFO - Starting established connection to tcp://10.3.88.11:37020
2024-02-01 05:57:28,955 - distributed.core - INFO - Event loop was unresponsive in Worker for 3.77s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 06:09:44,539 - distributed.core - INFO - Event loop was unresponsive in Worker for 98.95s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 06:09:44,539 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 98.95s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 30 conformers. 
Generated 22 conformers. 
Removed 19 conformers. 
Removed 14 conformers. 
Using force field
Optimising conformer:   0%|                              | 0/11 [00:00<?, ?it/s]Optimising conformer:   9%|██                    | 1/11 [00:01<00:11,  1.16s/it]Optimising conformer:  18%|████                  | 2/11 [00:02<00:11,  1.28s/it]Optimising conformer:  27%|██████                | 3/11 [00:03<00:10,  1.27s/it]Optimising conformer:  36%|████████              | 4/11 [00:05<00:09,  1.35s/it]Optimising conformer:  45%|██████████            | 5/11 [00:05<00:06,  1.08s/it]Optimising conformer:  55%|████████████          | 6/11 [00:06<00:05,  1.04s/it]Optimising conformer:  64%|██████████████        | 7/11 [00:07<00:03,  1.08it/s]Optimising conformer:  73%|████████████████      | 8/11 [00:08<00:02,  1.15it/s]Optimising conformer:  82%|██████████████████    | 9/11 [00:09<00:02,  1.08s/it]Optimising conformer:  91%|███████████████████  | 10/11 [00:10<00:01,  1.04s/it]Optimising conformer: 100%|█████████████████████| 11/11 [00:11<00:00,  1.03it/s]Optimising conformer: 100%|█████████████████████| 11/11 [00:11<00:00,  1.05s/it]
2024-02-01 06:10:48,740 - distributed.utils_perf - INFO - full garbage collection released 23.98 MiB from 310068 reference cycles (threshold: 9.54 MiB)
2024-02-01 06:10:48,740 [INFO] [utils_perf.py:198] distributed.utils_perf: full garbage collection released 23.98 MiB from 310068 reference cycles (threshold: 9.54 MiB)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/__init__.py:55: UserWarning: Dependency not satisfied, torchani.ase will not be available
  warnings.warn("Dependency not satisfied, torchani.ase will not be available")
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 06:11:02,635 [WARNING] [__init__.py:5] root: Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
TIME Completed the molecule generation in 78.3s.
TIME changed dir: 0.0s
failed to equip `nnpops` with error: No module named 'NNPOps'
Generated 20 conformers. 
Optimising conformer:   0%|                               | 0/8 [00:00<?, ?it/s][W BinaryOps.cpp:601] Warning: floor_divide is deprecated, and will be removed in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values.
To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor'). (function operator())
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
Optimising conformer:  12%|██▉                    | 1/8 [00:04<00:32,  4.69s/it]Optimising conformer:  25%|█████▊                 | 2/8 [00:06<00:18,  3.06s/it]Optimising conformer:  38%|████████▋              | 3/8 [00:21<00:42,  8.52s/it]Optimising conformer:  50%|███████████▌           | 4/8 [00:23<00:23,  5.84s/it]Optimising conformer:  62%|██████████████▍        | 5/8 [00:25<00:13,  4.64s/it]Optimising conformer:  75%|█████████████████▎     | 6/8 [00:27<00:07,  3.73s/it]Optimising conformer:  88%|████████████████████▏  | 7/8 [00:29<00:03,  3.16s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:31<00:00,  2.76s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:31<00:00,  3.97s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 06:12:12,374 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 3)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-02-01 06:12:16,912 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 3)

Removed 15 conformers. 
TIME Completed the molecule generation in 140.2s.
TIME changed dir: 0.0s
Generated 43 conformers. 
Removed 33 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/5 [00:00<?, ?it/s]Optimising conformer:  20%|████▌                  | 1/5 [00:04<00:19,  4.92s/it]Optimising conformer:  40%|█████████▏             | 2/5 [00:06<00:09,  3.20s/it]Optimising conformer:  60%|█████████████▊         | 3/5 [00:08<00:05,  2.65s/it]Optimising conformer:  80%|██████████████████▍    | 4/5 [00:10<00:02,  2.35s/it]Optimising conformer: 100%|███████████████████████| 5/5 [00:15<00:00,  3.28s/it]Optimising conformer: 100%|███████████████████████| 5/5 [00:15<00:00,  3.15s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 06:12:54,905 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 2)
 - Atom C (index 7)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-02-01 06:12:56,547 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 2)
 - Atom C (index 7)

TIME Completed the molecule generation in 105.2s.
TIME changed dir: 0.0s
Generated 49 conformers. 
Removed 44 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                              | 0/10 [00:00<?, ?it/s]Optimising conformer:  10%|██▏                   | 1/10 [00:03<00:31,  3.45s/it]Optimising conformer:  20%|████▍                 | 2/10 [00:06<00:25,  3.13s/it]Optimising conformer:  30%|██████▌               | 3/10 [00:09<00:21,  3.10s/it]Optimising conformer:  40%|████████▊             | 4/10 [00:13<00:22,  3.68s/it]Optimising conformer:  50%|███████████           | 5/10 [00:16<00:16,  3.36s/it]Optimising conformer:  60%|█████████████▏        | 6/10 [00:20<00:13,  3.43s/it]Optimising conformer:  70%|███████████████▍      | 7/10 [00:23<00:10,  3.37s/it]Optimising conformer:  80%|█████████████████▌    | 8/10 [00:27<00:07,  3.61s/it]Optimising conformer:  90%|███████████████████▊  | 9/10 [00:30<00:03,  3.21s/it]Optimising conformer: 100%|█████████████████████| 10/10 [00:35<00:00,  3.87s/it]Optimising conformer: 100%|█████████████████████| 10/10 [00:35<00:00,  3.54s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME Completed the molecule generation in 144.3s.
TIME changed dir: 0.0s
Generated 40 conformers. 
Removed 35 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/5 [00:00<?, ?it/s]Optimising conformer:  20%|████▌                  | 1/5 [00:05<00:21,  5.31s/it]Optimising conformer:  40%|█████████▏             | 2/5 [00:08<00:12,  4.08s/it]using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'

Optimising conformer:   0%|                               | 0/5 [00:00<?, ?it/s][AOptimising conformer:  60%|█████████████▊         | 3/5 [00:11<00:07,  3.78s/it]
Optimising conformer:  20%|████▌                  | 1/5 [00:04<00:18,  4.59s/it][A
Optimising conformer:  40%|█████████▏             | 2/5 [00:09<00:13,  4.56s/it][AOptimising conformer:  80%|██████████████████▍    | 4/5 [00:24<00:07,  7.26s/it]
Optimising conformer:  60%|█████████████▊         | 3/5 [00:12<00:08,  4.15s/it][A
Optimising conformer:  80%|██████████████████▍    | 4/5 [00:15<00:03,  3.69s/it][A
Optimising conformer: 100%|███████████████████████| 5/5 [00:20<00:00,  3.96s/it][AOptimising conformer: 100%|███████████████████████| 5/5 [00:20<00:00,  4.04s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
Optimising conformer: 100%|███████████████████████| 5/5 [00:37<00:00,  9.44s/it]Optimising conformer: 100%|███████████████████████| 5/5 [00:37<00:00,  7.57s/it]
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-02-01 06:16:07,999 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 3)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-02-01 06:16:09,789 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 3)

TIME Completed the molecule generation in 186.3s.
TIME changed dir: 0.0s
Generated 33 conformers. 
Removed 15 conformers. 
TIME Completed the molecule generation in 92.4s.
TIME changed dir: 0.0s
Generated 42 conformers. 
Removed 30 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                              | 0/18 [00:00<?, ?it/s]Optimising conformer:   6%|█▏                    | 1/18 [00:03<01:02,  3.70s/it]Optimising conformer:  11%|██▍                   | 2/18 [00:06<00:49,  3.12s/it]Optimising conformer:  17%|███▋                  | 3/18 [00:11<01:02,  4.18s/it]Optimising conformer:  22%|████▉                 | 4/18 [00:14<00:51,  3.69s/it]Optimising conformer:  28%|██████                | 5/18 [00:18<00:46,  3.57s/it]Optimising conformer:  33%|███████▎              | 6/18 [00:22<00:44,  3.72s/it]using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'

Optimising conformer:   0%|                              | 0/12 [00:00<?, ?it/s][AOptimising conformer:  39%|████████▌             | 7/18 [00:27<00:47,  4.36s/it]
Optimising conformer:   8%|█▊                    | 1/12 [00:05<01:01,  5.56s/it][AOptimising conformer:  44%|█████████▊            | 8/18 [00:31<00:41,  4.18s/it]
Optimising conformer:  17%|███▋                  | 2/12 [00:09<00:44,  4.45s/it][AOptimising conformer:  50%|███████████           | 9/18 [00:36<00:38,  4.26s/it]
Optimising conformer:  25%|█████▌                | 3/12 [00:11<00:32,  3.65s/it][A
Optimising conformer:  33%|███████▎              | 4/12 [00:14<00:26,  3.27s/it][AOptimising conformer:  56%|███████████▋         | 10/18 [00:41<00:37,  4.71s/it]
Optimising conformer:  42%|█████████▏            | 5/12 [00:17<00:22,  3.15s/it][AOptimising conformer:  61%|████████████▊        | 11/18 [00:46<00:32,  4.62s/it]
Optimising conformer:  50%|███████████           | 6/12 [00:21<00:20,  3.38s/it][A
Optimising conformer:  58%|████████████▊         | 7/12 [00:25<00:18,  3.64s/it][AOptimising conformer:  67%|██████████████       | 12/18 [00:51<00:29,  4.84s/it]Optimising conformer:  72%|███████████████▏     | 13/18 [00:56<00:24,  4.92s/it]
Optimising conformer:  67%|██████████████▋       | 8/12 [00:31<00:17,  4.48s/it][A
Optimising conformer:  75%|████████████████▌     | 9/12 [00:35<00:12,  4.23s/it][AOptimising conformer:  78%|████████████████▎    | 14/18 [01:03<00:22,  5.55s/it]
Optimising conformer:  83%|█████████████████▌   | 10/12 [00:41<00:09,  4.82s/it][AOptimising conformer:  83%|█████████████████▌   | 15/18 [01:07<00:15,  5.18s/it]Optimising conformer:  89%|██████████████████▋  | 16/18 [01:11<00:09,  4.57s/it]
Optimising conformer:  92%|███████████████████▎ | 11/12 [00:45<00:04,  4.49s/it][A
Optimising conformer: 100%|█████████████████████| 12/12 [00:50<00:00,  4.78s/it][AOptimising conformer: 100%|█████████████████████| 12/12 [00:50<00:00,  4.24s/it]
Optimising conformer:  94%|███████████████████▊ | 17/18 [01:18<00:05,  5.53s/it]Optimising conformer: 100%|█████████████████████| 18/18 [01:23<00:00,  5.11s/it]Optimising conformer: 100%|█████████████████████| 18/18 [01:23<00:00,  4.61s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 06:30:28,567 - distributed.core - INFO - Event loop was unresponsive in Nanny for 62.69s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 06:30:28,571 - distributed.core - INFO - Event loop was unresponsive in Worker for 64.72s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 06:30:28,571 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 64.72s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 06:45:55,984 - distributed.core - INFO - Event loop was unresponsive in Worker for 6.21s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 06:45:55,984 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 6.21s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 06:52:12,640 - distributed.core - INFO - Event loop was unresponsive in Worker for 6.97s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 06:52:12,640 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 6.97s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME Completed the molecule generation in 127.7s.
TIME Completed the molecule generation in 140.8s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 43 conformers. 
Generated 47 conformers. 
Removed 35 conformers. 
Removed 36 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/8 [00:00<?, ?it/s]Optimising conformer:  12%|██▉                    | 1/8 [00:03<00:27,  3.99s/it]Optimising conformer:  25%|█████▊                 | 2/8 [00:21<01:13, 12.20s/it]Optimising conformer:  38%|████████▋              | 3/8 [00:27<00:44,  8.97s/it]Optimising conformer:  50%|███████████▌           | 4/8 [00:32<00:29,  7.43s/it]Optimising conformer:  62%|██████████████▍        | 5/8 [00:36<00:18,  6.33s/it]Optimising conformer:  75%|█████████████████▎     | 6/8 [00:43<00:13,  6.58s/it]Optimising conformer:  88%|████████████████████▏  | 7/8 [00:56<00:08,  8.57s/it]Optimising conformer: 100%|███████████████████████| 8/8 [01:01<00:00,  7.44s/it]Optimising conformer: 100%|███████████████████████| 8/8 [01:01<00:00,  7.66s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 07:16:46,597 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Bonds with undefined stereochemistry are:
 - Bond 8 (atoms 8-9 of element (C-N)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-02-01 07:16:48,305 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Bonds with undefined stereochemistry are:
 - Bond 8 (atoms 8-9 of element (C-N)

TIME Completed the molecule generation in 194.9s.
TIME changed dir: 0.0s
Generated 34 conformers. 
Removed 24 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                              | 0/10 [00:00<?, ?it/s]Optimising conformer:  10%|██▏                   | 1/10 [00:04<00:40,  4.45s/it]Optimising conformer:  20%|████▍                 | 2/10 [00:07<00:30,  3.82s/it]Optimising conformer:  30%|██████▌               | 3/10 [00:10<00:24,  3.44s/it]Optimising conformer:  40%|████████▊             | 4/10 [00:14<00:21,  3.59s/it]Optimising conformer:  50%|███████████           | 5/10 [00:18<00:19,  3.83s/it]Optimising conformer:  60%|█████████████▏        | 6/10 [00:22<00:14,  3.68s/it]Optimising conformer:  70%|███████████████▍      | 7/10 [00:25<00:10,  3.47s/it]Optimising conformer:  80%|█████████████████▌    | 8/10 [00:28<00:06,  3.36s/it]Optimising conformer:  90%|███████████████████▊  | 9/10 [00:31<00:03,  3.42s/it]Optimising conformer: 100%|█████████████████████| 10/10 [00:35<00:00,  3.30s/it]Optimising conformer: 100%|█████████████████████| 10/10 [00:35<00:00,  3.50s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
TIME Completed the molecule generation in 119.2s.
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                              | 0/11 [00:00<?, ?it/s]Optimising conformer:   9%|██                    | 1/11 [00:05<00:56,  5.68s/it]Optimising conformer:  18%|████                  | 2/11 [00:09<00:43,  4.85s/it]Optimising conformer:  27%|██████                | 3/11 [00:13<00:34,  4.35s/it]Optimising conformer:  36%|████████              | 4/11 [00:16<00:25,  3.66s/it]Optimising conformer:  45%|██████████            | 5/11 [00:19<00:21,  3.52s/it]Optimising conformer:  55%|████████████          | 6/11 [00:22<00:16,  3.24s/it]Optimising conformer:  64%|██████████████        | 7/11 [00:27<00:15,  3.77s/it]Optimising conformer:  73%|████████████████      | 8/11 [00:31<00:12,  4.07s/it]Optimising conformer:  82%|██████████████████    | 9/11 [00:35<00:07,  3.96s/it]Optimising conformer:  91%|███████████████████  | 10/11 [00:40<00:04,  4.35s/it]Optimising conformer: 100%|█████████████████████| 11/11 [00:45<00:00,  4.35s/it]Optimising conformer: 100%|█████████████████████| 11/11 [00:45<00:00,  4.11s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 08:01:20,360 - distributed.core - INFO - Event loop was unresponsive in Worker for 3.61s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 08:01:20,360 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 3.61s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 08:01:23,393 - distributed.core - INFO - Event loop was unresponsive in Worker for 3.04s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 08:01:23,393 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 3.04s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 08:03:10,037 - distributed.core - INFO - Event loop was unresponsive in Worker for 3.78s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 08:03:10,037 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 3.78s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 08:35:38,047 - distributed.worker - ERROR - Worker stream died during communication: tcp://10.3.89.93:44896
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.93:44896 after 30 s
2024-02-01 08:35:38,047 [ERROR] [worker.py:2115] distributed.worker: Worker stream died during communication: tcp://10.3.89.93:44896
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.93:44896 after 30 s
2024-02-01 08:36:08,116 - distributed.worker - ERROR - Worker stream died during communication: tcp://10.3.89.95:36040
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.95:36040 after 30 s
2024-02-01 08:36:08,116 [ERROR] [worker.py:2115] distributed.worker: Worker stream died during communication: tcp://10.3.89.95:36040
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.95:36040 after 30 s
2024-02-01 08:36:39,255 - distributed.worker - ERROR - Worker stream died during communication: tcp://10.3.89.93:44896
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.93:44896 after 30 s
2024-02-01 08:36:39,255 [ERROR] [worker.py:2115] distributed.worker: Worker stream died during communication: tcp://10.3.89.93:44896
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.93:44896 after 30 s
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME Completed the molecule generation in 400.2s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 6 conformers. 
Removed 4 conformers. 
Generated 23 conformers. 
Removed 15 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/2 [00:00<?, ?it/s]Optimising conformer:  50%|███████████▌           | 1/2 [00:04<00:04,  4.34s/it]Optimising conformer: 100%|███████████████████████| 2/2 [00:07<00:00,  3.51s/it]Optimising conformer: 100%|███████████████████████| 2/2 [00:07<00:00,  3.64s/it]
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/8 [00:00<?, ?it/s]/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
Optimising conformer:  12%|██▉                    | 1/8 [00:06<00:46,  6.58s/it]Optimising conformer:  25%|█████▊                 | 2/8 [00:08<00:24,  4.11s/it]Optimising conformer:  38%|████████▋              | 3/8 [00:14<00:24,  4.83s/it]/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
Optimising conformer:  50%|███████████▌           | 4/8 [00:19<00:18,  4.67s/it]Optimising conformer:  62%|██████████████▍        | 5/8 [00:23<00:14,  4.70s/it]Optimising conformer:  75%|█████████████████▎     | 6/8 [00:29<00:10,  5.09s/it]Optimising conformer:  88%|████████████████████▏  | 7/8 [00:33<00:04,  4.70s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:37<00:00,  4.56s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:37<00:00,  4.73s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
TIME Completed the molecule generation in 79.1s.
TIME changed dir: 0.0s
Generated 49 conformers. 
Removed 45 conformers. 
TIME Completed the molecule generation in 128.5s.
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/4 [00:00<?, ?it/s]Optimising conformer:  25%|█████▊                 | 1/4 [00:06<00:18,  6.03s/it]Optimising conformer:  50%|███████████▌           | 2/4 [00:10<00:10,  5.14s/it]Optimising conformer:  75%|█████████████████▎     | 3/4 [00:13<00:03,  3.99s/it]Optimising conformer: 100%|███████████████████████| 4/4 [00:17<00:00,  4.00s/it]Optimising conformer: 100%|███████████████████████| 4/4 [00:17<00:00,  4.29s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 09:05:10,629 - distributed.core - INFO - Event loop was unresponsive in Worker for 3.39s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 09:05:10,629 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 3.39s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME Completed the molecule generation in 175.4s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 8 conformers. 
Removed 3 conformers. 
Generated 10 conformers. 
Removed 7 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/3 [00:00<?, ?it/s]Optimising conformer:  33%|███████▋               | 1/3 [00:02<00:05,  2.77s/it]Optimising conformer:  67%|███████████████▎       | 2/3 [00:06<00:03,  3.19s/it]Optimising conformer: 100%|███████████████████████| 3/3 [00:07<00:00,  2.45s/it]Optimising conformer: 100%|███████████████████████| 3/3 [00:07<00:00,  2.60s/it]
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/5 [00:00<?, ?it/s]/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
Optimising conformer:  20%|████▌                  | 1/5 [00:04<00:17,  4.39s/it]Optimising conformer:  40%|█████████▏             | 2/5 [00:05<00:08,  2.71s/it]/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
Optimising conformer:  60%|█████████████▊         | 3/5 [00:08<00:05,  2.84s/it]Optimising conformer:  80%|██████████████████▍    | 4/5 [00:10<00:02,  2.51s/it]Optimising conformer: 100%|███████████████████████| 5/5 [00:13<00:00,  2.54s/it]Optimising conformer: 100%|███████████████████████| 5/5 [00:13<00:00,  2.70s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 09:24:02,781 - distributed.worker - WARNING - Compute Failed
Key:       evaluate-dbf68ec6-53e1-420b-9565-b8c75c2e64eb
Function:  evaluate
args:      (<rdkit.Chem.rdchem.Mol object at 0x2ba4d91ecdb0>, 6, '[H]c1nc([H])c(N2C(=O)C(=O)N([C@@]3([H])C([H])([H])[C@]3([H])c3c([H])c([H])c([H])c([H])c3[H])C2=O)c([H])c1[H]', '/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rec_final.pdb', '/nobackup/nmb1063/sars-fegrow/gnina')
kwargs:    {}
Exception: "Exception('No Conformers')"

2024-02-01 09:24:02,781 [WARNING] [worker.py:2352] distributed.worker: Compute Failed
Key:       evaluate-dbf68ec6-53e1-420b-9565-b8c75c2e64eb
Function:  evaluate
args:      (<rdkit.Chem.rdchem.Mol object at 0x2ba4d91ecdb0>, 6, '[H]c1nc([H])c(N2C(=O)C(=O)N([C@@]3([H])C([H])([H])[C@]3([H])c3c([H])c([H])c([H])c([H])c3[H])C2=O)c([H])c1[H]', '/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rec_final.pdb', '/nobackup/nmb1063/sars-fegrow/gnina')
kwargs:    {}
Exception: "Exception('No Conformers')"

TIME Completed the molecule generation in 57.0s.
TIME changed dir: 0.0s
Generated 8 conformers. 
Removed 0 conformers. 
TIME Completed the molecule generation in 72.8s.
TIME changed dir: 0.0s
Generated 23 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
Removed 23 conformers. 
Warning: no conformers so cannot optimise_in_receptor. Ignoring.
TIME changed dir: 0.0s
failed to equip `nnpops` with error: No module named 'NNPOps'
Generated 46 conformers. 
Optimising conformer:   0%|                               | 0/8 [00:00<?, ?it/s]2024-02-01 09:24:18,993 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 29)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-02-01 09:24:21,466 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 29)

Optimising conformer:  12%|██▉                    | 1/8 [00:05<00:36,  5.23s/it]Optimising conformer:  25%|█████▊                 | 2/8 [00:07<00:21,  3.55s/it]Optimising conformer:  38%|████████▋              | 3/8 [00:11<00:18,  3.69s/it]Optimising conformer:  50%|███████████▌           | 4/8 [00:13<00:11,  2.89s/it]Optimising conformer:  62%|██████████████▍        | 5/8 [00:14<00:07,  2.50s/it]Optimising conformer:  75%|█████████████████▎     | 6/8 [00:19<00:06,  3.09s/it]Optimising conformer:  88%|████████████████████▏  | 7/8 [00:21<00:02,  2.68s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:24<00:00,  2.97s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:24<00:00,  3.08s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
Removed 36 conformers. 
TIME Completed the molecule generation in 74.2s.
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                              | 0/10 [00:00<?, ?it/s]Optimising conformer:  10%|██▏                   | 1/10 [00:04<00:38,  4.31s/it]Optimising conformer:  20%|████▍                 | 2/10 [00:07<00:28,  3.53s/it]Optimising conformer:  30%|██████▌               | 3/10 [00:12<00:28,  4.11s/it]Optimising conformer:  40%|████████▊             | 4/10 [00:14<00:21,  3.61s/it]Optimising conformer:  50%|███████████           | 5/10 [00:17<00:15,  3.15s/it]Optimising conformer:  60%|█████████████▏        | 6/10 [00:20<00:13,  3.35s/it]Optimising conformer:  70%|███████████████▍      | 7/10 [00:25<00:11,  3.83s/it]Optimising conformer:  80%|█████████████████▌    | 8/10 [00:28<00:06,  3.43s/it]Optimising conformer:  90%|███████████████████▊  | 9/10 [00:34<00:04,  4.21s/it]Optimising conformer: 100%|█████████████████████| 10/10 [00:39<00:00,  4.53s/it]Optimising conformer: 100%|█████████████████████| 10/10 [00:39<00:00,  3.96s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
slurmstepd: error: *** JOB 19912145 ON sb039 CANCELLED AT 2024-02-01T09:54:42 ***
